src/Microsoft.AspNet.Mvc.ModelBinding/Binders/CompositeModelBinder.cs

```diff
@@ -129,12 +132,6 @@ private static bool IsBindingAtRootOfObjectGraph(ModelBindingContext bindingCont
                 PropertyFilter = oldBindingContext.PropertyFilter,
             };
 
-            // validation is expensive to create, so copy it over if we can
```

Comment:
Nope, I don't think it was solving anything, perf is something we can take care of if we find it is a hotspot. What security concern do you think can happen here?